docs-src/tut-7-rpc.scrbl

```diff
@@ -183,6 +210,144 @@ invoke our @pyin{main} function.
+your @tt{rps-python} directory and type @tt{./reach tls}:
+
+@verbatim{
+$ ./reach tls
+*** Warning! Certificates generated by `reach tls` are only suitable ***
+    for development purposes and must not be used in production!
+
+Scaffolding auto-generated TLS certificate + key under ./tls directory...
+(When prompted for a pass phrase, please enter between 4 and 1024 characters)
+
+Generating a RSA private key
+....................................................++++
+.................++++
+writing new private key to './tls/reach-server.key'
+Enter PEM pass phrase:
+Verifying - Enter PEM pass phrase:
+-----
+Signature ok
+subject=CN = reach-server
+Getting Private key
+Enter pass phrase for ./tls/reach-server.key:
+  ...done
+
+When invoking `reach server` you MUST supply the pass phrase you just
+used to generate your certificate as an environment variable, e.g.:
+
+  $ REACH_RPC_TLS_PASSPHRASE=ENTER-PASSPHRASE-HERE \
+    REACH_RPC_KEY=YOUR-PRESHARED-KEY \
+    ./reach server
+
+./tls
+├── reach-server.crt
+├── reach-server.csr
+├── reach-server.ext
+└── reach-server.key
+
+0 directories, 4 files
+}
+
+@(hrule)
+
+With those created, now we launch the server:
+
+@verbatim{
+$  REACH_RPC_TLS_PASSPHRASE=ENTER-PASSPHRASE-HERE \
+   REACH_RPC_KEY=YOUR-PRESHARED-KEY \
+   ./reach server
+
+===============================
+using rpc key:
+REACH_RPC_KEY=YOUR-PRESHARED-KEY
+===============================
+
+Verifying knowledge assertions
+Verifying for generic connector
+  Verifying when ALL participants are honest
+  Verifying when NO participants are honest
+  Verifying when ONLY "Alice" is honest
+  Verifying when ONLY "Bob" is honest
+Checked 93 theorems; No failures!
+Creating network "rps-python_default" with the default driver
+Creating rps-python_ethereum-devnet_1 ... done
+
+> @"@reach-sh/rpc-server@" app /app
+> node --experimental-modules --unhandled-rejections=strict index.mjs
+
+(node:26) ExperimentalWarning: The ESM module loader is experimental.
+}
+
+The server is now awaiting connections.
+
+@(hrule)
+
+Open another terminal in your @tt{rps-python} directory and type the following
+to play a game of @|RPS|:
+
+@margin-note{
+Alternatively, if you would prefer to supply configuration as arguments to the
+@pyin{mk_rpc} function at runtime instead of using environment variables, this
+also works:
+
+@verbatim{
+rpc, rpc_callbacks = \
+    mk_rpc(dict(
+  host='127.0.0.1',
+  port='3000',
+  verify='0',
+  timeout='7',
+  key='YOUR-PRESHARED-KEY',
+))
+}
+
+
+You may choose whichever best suits your use case.
+
+Combining both methods is also possible, but settings supplied as arguments to
+@pyin{mk_rpc} are given priority over settings defined as environment
+variables.
+
+Read @seclink{ref-backends-rpc-opts} for more details.
+}
+
+@verbatim{
+$ . ./venv/bin/activate
+$ REACH_RPC_SERVER=127.0.0.1 \
+  REACH_RPC_PORT=3000 \
+  REACH_RPC_KEY=YOUR-PRESHARED-KEY \
```

Comment:
You should use a particular key throughout this whole example and just mention that you can make it whatever you want. It should be something obviously bad like `opensesame`